src/_pytest/main.py

```diff
@@ -63,9 +63,7 @@ def pytest_addoption(parser: Parser) -> None:
             "*.egg",
             ".*",
             "_darcs",
-            "build",
```

Comment:
I think that the description string above should mention that these are still conditionally excluded by default.